query-service/src/integrationTest/java/org/hypertrace/core/query/service/QueryServiceTestUtils.java

```diff
@@ -63,4 +62,9 @@ public static ColumnIdentifier createColumnIdentifier(String columnName) {
   public static ColumnIdentifier createColumnIdentifier(String columnName, String alias) {
     return ColumnIdentifier.newBuilder().setColumnName(columnName).setAlias(alias).build();
   }
+
+  public static Expression.Builder createColumnExpression(String columnName) {
```

Comment:
There's two query request utils. The one in the api package was deprecated (because a util doesn't belong to the API), and the methods used by query service itself were copied into `org/hypertrace/core/query/service/QueryRequestUtil.java`